src/Model/NumericalModel.f90

```diff
@@ -439,4 +440,43 @@ function GetNumericalModelFromList(list, idx) result(res)
     return
   end function GetNumericalModelFromList
 
+  !> @brief Load input context for supported package
+  !<
+  subroutine load_input_context(this, filtyp, modelname, pkgname, inunit, iout, &
+                                ipaknum)
+    ! -- modules
+    use IdmMf6FileLoaderModule, only: input_load
+    ! -- dummy
+    class(NumericalModelType) :: this
+    character(len=*), intent(in) :: filtyp
+    character(len=*), intent(in) :: modelname
+    character(len=*), intent(in) :: pkgname
+    integer(I4B), intent(in) :: inunit
+    integer(I4B), intent(in) :: iout
+    integer(I4B), optional, intent(in) :: ipaknum
+    ! -- local
+! ------------------------------------------------------------------------------
+    !
+    ! -- only load if there is a file to read
+    if (inunit <= 0) return
+    !
+    ! -- Load model package input to input context
+    select case (filtyp)
+    case ('DIS6')
+      call input_load('DIS6', 'GWF', 'DIS', modelname, pkgname, inunit, iout)
+    case ('DISU6')
+      call input_load('DISU6', 'GWF', 'DISU', modelname, pkgname, inunit, iout)
+    case ('DISV6')
+      call input_load('DISV6', 'GWF', 'DISV', modelname, pkgname, inunit, iout)
+    case ('DSP6')
+      call input_load('DSP6', 'GWT', 'DSP', modelname, pkgname, inunit, iout)
+    case ('NPF6')
+      call input_load('NPF6', 'GWF', 'NPF', modelname, pkgname, inunit, iout)
```

Comment:
I'm not sure DSP and NPF should be here in NumericalModel as these packages are specific to GWT and GWF, respectively.  I suspect this is a temporary location.

Reply:
My focus was on minimizing code impact as I do see it as a temporary move but you make a good point and the changes will still be minimal- commit coming.

Reply:
I was thinking that with consensus it could be immediately merged but I'm open either way.  Do you have a preference?